src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.component.js

```diff
@@ -30,16 +32,50 @@ const EnrollmentPagePlain = ({ classes, enrollmentPageStatus }) => (<>
         }
 
         {
-            enrollmentPageStatus === enrollmentPageStatuses.LOADING &&
-                <div className={classes.loadingMask}>
-                    <LoadingMaskForPage />
+            enrollmentPageStatus === enrollmentPageStatuses.MISSING_PROGRAM_SELECTION &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('Choose program to view more information')}
+            </IncompleteSelectionsMessage>
+        }
+
+        {
+            enrollmentPageStatus === enrollmentPageStatuses.MISSING_PROGRAM_CATEGORIES_SELECTION &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('MISSING CATEGORIES NEEDS BETTER TEXT')}
+            </IncompleteSelectionsMessage>
+        }
+
+        {
+            enrollmentPageStatus === enrollmentPageStatuses.MISSING_ENROLLMENT_SELECTION &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('Choose enrollment to view more information')}
+            </IncompleteSelectionsMessage>
+
+        }
+
+        {
+            enrollmentPageStatus === enrollmentPageStatuses.TRACKER_PROGRAM_WITH_ZERO_ENROLLMENTS_SELECTED &&
+            <IncompleteSelectionsMessage>
+                {i18n.t('There are no enrollments for this program.')}
+            </IncompleteSelectionsMessage>
+        }
+
+        {
+            // todo need to make "here" a link
+            enrollmentPageStatus === enrollmentPageStatuses.EVENT_PROGRAM_SELECTED &&
+            <IncompleteSelectionsMessage>
+                <div style={{ textAlign: 'center' }}>
+                    {i18n.t('You selected an event program. Event do not have enrollments.')}
```

Comment:
Suggetion from Karoline => "You selected an Event program. Event programs do not have  tracked entities and enrollments. "